Source/WebKit/UIProcess/ProvisionalPageProxy.cpp

```diff
@@ -191,7 +191,7 @@ void ProvisionalPageProxy::cancel()
 
 void ProvisionalPageProxy::initializeWebPage(RefPtr<API::WebsitePolicies>&& websitePolicies)
 {
-    m_drawingArea = m_page->pageClient().createDrawingAreaProxy();
+    m_drawingArea = m_page->pageClient().createDrawingAreaProxy(m_process);
```

Comment:
`m_process.copyRef()` to follow our new safe smart pointer idioms. We need to make sure we have a smart pointer on the stack to the process we're passing in parameter.